packages/databricks-vscode/src/run/WorkflowRunner.ts

```diff
@@ -125,14 +126,38 @@ export class WorkflowRunner implements Disposable {
                     }
             }
         });
+
         try {
-            if (await isNotebook(program)) {
+            const notebookType = await isNotebook(program);
+            if (notebookType) {
+                let remoteFilePath: string =
+                    syncDestination.localToRemoteNotebook(program).path;
+                if (workspaceConfigs.enableFilesInWorkspace) {
```

Comment:
This branch should look at the sync destination path and not the setting. The user might still have a Repo path after changing the setting.